sdk/storage/azblob/blob/client.go

```diff
@@ -38,9 +38,12 @@ func NewClient(blobURL string, cred azcore.TokenCredential, options *ClientOptio
 	authPolicy := shared.NewStorageChallengePolicy(cred)
 	conOptions := shared.GetClientOptions(options)
 	conOptions.PerRetryPolicies = append(conOptions.PerRetryPolicies, authPolicy)
-	pl := runtime.NewPipeline(exported.ModuleName, exported.ModuleVersion, runtime.PipelineOptions{}, &conOptions.ClientOptions)
 
-	return (*Client)(base.NewBlobClient(blobURL, pl, &cred)), nil
+	azClient, err := azcore.NewClient("blob.Client", exported.ModuleVersion, runtime.PipelineOptions{}, &conOptions.ClientOptions)
```

Comment:
Earlier today I noticed that this incorrectly sets the module name to `blob` in the telemetry string.  I've fixed this in `azcore@v1.6.1` released today.  Can you please update to that version and change all of these strings to `module/package.Client`.  So for this one it would be `azblob/blob.Client`.  No need to change the string for `azblob.Client`.